### PR TITLE
Add info section to budgets in the selecting phase

### PR DESCRIPTION
**Pull request by taitus**
## References
- This is part of pull request #4198

## Objectives
Add supports info section on budgets index page.

## Visual Changes
Captura de pantalla 2021-05-10 a las 17 18 50

## Notes
Merge after #4507

### CODE REVIEW

**Review by Senen — CHANGES_REQUESTED** [review]
Hi @taitus, I found some minor things. Let me know what you think. ☺️

File: app/components/budgets/supports_info_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<% if budget.selecting? %>
+  <div class="supports-info text-center">
+    <div class="callout">
+      <h2><%= sanitize(t("budgets.index.supports_info.title")) %></h2>
+
+      <%= image_tag("budgets/budgets_support.png", alt: "", class: "margin") %>
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/budgets/supports_info_component.html.erb:6]: @Senen IMHO this is a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/) :thinking:, so leaving the alt attribute blank is fine.

We could also remove this line and add the image using CSS pseudoelements like `.supports-info .callout h2::after` :thinking:. That way other CONSUL installations could customize the image without changing ERB files.

[the remark this answers is not in the document]
Comment (Senen) [reply app/components/budgets/supports_info_component.html.erb:6]: It's ok with me. I was worried about what accessibility validators would say, but I guess you already know that there will be no problem. 👍🏼

File: app/components/budgets/investments_list_component.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <% if investments.any? %>
-  <section class="investments-list">
+  <section id="investments-list" class="investments-list">
```
[the remark this answers is not in the document]
Comment (Senen) [reply app/components/budgets/investments_list_component.html.erb:2]: I was wrong, the id is needed [here](https://github.com/consul/consul/pull/4508/files#diff-a32c142943c511497cdad9a4143e0439b2932645407e93647d2b004999abad42R29).

**Review by javierm — APPROVED** [review]

**Pull request merged**